public/components/CampaignList/CampaignList.js

```diff
@@ -11,6 +11,40 @@ class CampaignList extends React.Component {
     campaigns: []
   };
 
+  state = {
+    sortOrder: {
+      'name': true, //false = DESC, true = ASC
+      'type': false,
+      'status': false,
+      'actualValue': false,
+      'startDate': false,
+      'endDate': false
+    },
+    headersCssClasses: {
+      'name': 'campaign-list__header-order--desc name',
```

Comment:
I know that you prefer to have one css class per tag @clloyd. Do you think that's an issue here?